gammapy/background/tests/test_models.py

```diff
@@ -43,3 +50,93 @@ def test_y_model(self):
         model = self.model.y_model(-30)
         assert isinstance(model, Gaussian1D)
         assert_allclose(model.parameters, [0, -1, 0.4])
+
+
+def test_wcs_object():
+    bins_x = Angle([1., 2., 3., 4.], 'degree')
+    bins_y = Angle([-1.5, 0., 1.5], 'degree')
+    wcs = make_linear_wcs_from_bin_edges_arrays("X", "Y", bins_x, bins_y)
+    nbins_x = len(bins_x) - 1
+    nbins_y = len(bins_y) - 1
+    reco_bins_x, reco_bins_y = make_linear_bin_edges_arrays_from_wcs(wcs,
```

Comment:
As you can see here, this utility function name is a bit too long.
Maybe `linear_array_to_wcs` and `linear_wcs_to_array`?